pkg/remotedialer/session.go

```diff
@@ -71,14 +71,11 @@ func (s *Session) startPings() {
 	go func() {
 		defer s.pingWait.Done()
 
-		t := time.NewTicker(PingWriteInterval)
-		defer t.Stop()
-
 		for {
 			select {
 			case <-ctx.Done():
 				return
-			case <-t.C:
+			case <-time.After(PingWriteInterval):
```

Comment:
time.After needs to be used sparingly and not in a loop.  According to the [godocs](https://godoc.org/time#After)

```
The underlying Timer is not recovered by the garbage collector until the timer fires. If efficiency is a concern, use NewTimer instead and call Timer.Stop if the timer is no longer needed. 
```